scripts/deploy/govcms-pre-deploy-db-update

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# Pre-deploy updb
+#
+# This will perform an drush updb on an environment when
+# the environment has the GOVCMS_DEPLOY_PRE_UPDB variable defined.
+#
+# In some use cases databaes updates need to be performed early in
+# the deploy process, even before the site has been bootstrapped
+# (for example with drush status). This provides the deploy agent
+# the ability to perform updates early in the automated flow.
+#
+# The ansible deploy process will add GOVCMS_DEPLOY_PRE_UPDB prior to
+# triggering the deploy to allow this to run properly.
+#
+
+LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
+GOVCMS_DEPLOY_PRE_UPDB=${GOVCMS_DEPLOY_PRE_UPDB:-false}
+
+echo "GovCMS Deploy :: Pre-deploy database updates"
+
+if [ "$GOVCMS_DEPLOY_PRE_UPDB" = false ]; then
+  # A pre-deploy update is not required so return early.
+  echo "[skip]: Environment variable not set"
```

Comment:
can we please change this message to have more context. Maybe something like "Skipping pre-deployment db update" or something along these lines.